apps/desktop-wallet/src/components/WalletSwitcher.tsx

```diff
@@ -71,7 +71,9 @@ const WalletSwitcher = ({ onUnlock }: WalletSwitcherProps) => {
         if (location.pathname !== nextPageLocation) navigate(nextPageLocation)
       }
     })
-    if (passphrase) setPassphrase('')
+
+    setPassphrase('')
+    password = ''
```

Comment:
Reassigning empty strings to function parameters is an approach Metamask is using: https://github.com/MetaMask/metamask-mobile/blob/c575f53/app/core/Authentication/Authentication.ts#L156-L164